### ISSUE TITLE
[BUG] World hopper tries to click world through bottom of menu

### ISSUE THREAD

**Issue by gavinnn101** [powbot/issues#140]
**Describe the bug**
tl;dr - world hopper doesn't count the bottom of the menu as blocking sight of the world it wants to hop to.
When trying to world hop it will scroll down and then start trying to click the world it wants to hop to through the bottom of the menu. You can see this in the gif: https://i.gyazo.com/b2c90e4c404fdf862bf552eed8a680f7.mp4

**To Reproduce/Line of code causing issue**
```
if (Worlds.stream().id(worlds[worldIndex]).first().hop()) {
    Condition.wait(() -> Worlds.current().getNumber() == worlds[worldIndex], 200, 20);
}
```

**Expected behavior**
Scroll down a bit more so the world is visible, then click.

**Screenshots**
https://i.gyazo.com/b2c90e4c404fdf862bf552eed8a680f7.mp4

**Emulator or device (please complete the following information):**
 - Device/Emulator: [Bluestacks]
 - OS: [Android Pie]
 - Arch: [ ARM64]

**Comment by ghost** [comment]
@gavinnn101 just an observation as well, make sure to do a .valid check on it first. 

```
var hopWorld = Worlds.stream().id(worlds[worldIndex]).first();
if (hopWorld.valid() && hopWorld.hop()) {
    Condition.wait(() -> Worlds.current().getNumber() == worlds[worldIndex], 200, 20);
}
```

It _could_ potentially fix your issue

**Comment by gavinnn101** [comment]
> @gavinnn101 just an observation as well, make sure to do a .valid check on it first.
> 
> ```
> var hopWorld = Worlds.stream().id(worlds[worldIndex]).first();
> if (hopWorld.valid() && hopWorld.hop()) {
>     Condition.wait(() -> Worlds.current().getNumber() == worlds[worldIndex], 200, 20);
> }
> ```
> 
> It _could_ potentially fix your issue

Appreciate the tip! I just tried this and it actually worked for the first 2 world hops and then got stuck trying to click the 3rd world through the bottom bar again instead of scrolling to get the world visible.

I don't know if it's important but I notice it clicks the bottom bar and a `Switch` option comes up like it'll let it world hop but then I confirmed manually clicking `Switch` doesnt world hop. I'm assuming because it doesn't give a world in the option. see screenshots:

Regular hop:
![image](https://user-images.githubusercontent.com/41026935/197409668-5b6354fb-f18c-43d3-8f9c-31b00bcaa1e9.png)

broken hop attempt:
![image](https://user-images.githubusercontent.com/41026935/197409692-536eae38-5c01-4c46-b422-f4963f537394.png)

**Comment by constt** [comment]
@gavinnn101 still an issue?
